### PR TITLE
Fix #249: Make sidebar in dashboard collapse at screensize<992px.

### LINKED ISSUE #249: Sidebar on Dashboard view is not working properly for Mobile view.
Currently sidebar is collapsing at ~640px but it should be collapsed at screensize<992 px.

### PR DESCRIPTION
**Pull request by MAKOSCAFEE**
This PR aims at making the sidebar in dashboard collapse for medium screensize(992px) and below.

### CODE REVIEW

**Comment by aka-jain** [comment]
@MAKOSCAFEE Can you do the same for bar icon in header too?
Header file is sim-header and you just need to change the class for icon as you did with sidebar.

**Comment by MAKOSCAFEE** [comment]
Hi @aka-jain, Sure

File: frontend/src/views/web/partials/sim-header.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
                 <li class="text-med-gray"><strong></strong>{{pageTitle}}</li>
             </ul>
             <ul class="right">
-                <li><a class="waves-effect waves-dark med-link ev-header-overflow" ui-sref="home"><i class="fa fa-home"></i> <span class="hide-on-small-only">Home</span></a></li>
+                <li><a class="waves-effect waves-dark med-link ev-header-overflow" ui-sref="home"><i class="fa fa-home"></i> <span class="hide-on-med-and-down">Home</span></a></li>
```
Comment (aka-jain) [inline frontend/src/views/web/partials/sim-header.html]: @MAKOSCAFEE No, we need to change bar icon only.

Comment (aka-jain) [inline frontend/src/views/web/partials/sim-header.html]: Just add hide-on-large-only to bar icon. See line no. 4

Comment (MAKOSCAFEE) [inline frontend/src/views/web/partials/sim-header.html]: Ok .. sorry.

**Comment by aka-jain** [comment]
Thanks @MAKOSCAFEE . 
Merging it.

**Pull request merged**